<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Lab2 Vlasenko Mykola</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTables">
     <attribute name="title">
      <string>Tables</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxTables">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>311</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSelectTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Table:</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewTables">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>720</width>
        <height>440</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabViews">
     <attribute name="title">
      <string>Views</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxViews">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>531</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSelectView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select View:</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewViews">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>109</y>
        <width>720</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelViewSQLText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SQL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSQLTextViews">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>681</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabParamViews">
     <attribute name="title">
      <string>ParamViews</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxParamViews">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>531</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSelectParamView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Parametrized View:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelParamViewSQLText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SQL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSQLTextParamViews">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>681</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewParamViews">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>720</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open DB...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
